CSR-ATTESTATION-2023.asn

```diff
@@ -14,28 +14,31 @@ Certificate
      mechanisms(5) pkix(7) id-mod(0) id-mod-pkix1-explicit-02(51)}
 
 
-EXTENSION, ATTRIBUTE, AttributeSet{}, SingleAttribute{}, id-pkix 
+EXTENSION, ATTRIBUTE, AttributeSet{}, SingleAttribute{}
     FROM PKIX-CommonTypes-2009 -- from [RFC5912]
     { iso(1) identified-organization(3) dod(6) internet(1) security(5)
       mechanisms(5) pkix(7) id-mod(0) id-mod-pkixCommon-02(57) }
 
-id-aa
```

Comment:
You should put this back. The only reason I removed it was that I did not have SecureMimeMessageV3dot1 in the collection of ASN.1 modules I use to test compile. It is there though.